### ISSUE TITLE
Question about reward normalization in evaluation

### ISSUE THREAD

**Issue by cheryyunl** [YangRui2015/RiC#16]
Hi! Thanks for this amazing work and code open-sourcing! 
Quick question - we're evaluating our SFT models and noticed our obtained reward values are quite different from Figure 4(a) in your paper. Since you mentioned "normalized rewards" in the figure caption, could you share what normalization method was used?

Also, wondering if you used the same dataset combo as us (helpful-base, helpful-online, harmless-base from hh-rlhf, without red-team data)? https://huggingface.co/datasets/Anthropic/hh-rlhf

Thanks in advance! 🙏

**Comment by YangRui2015** [comment]
Thank you for your interest in our work! Yes, we use some normalization configuration from the training dataset. I used the following configuration (mean and std) for the harmless (R1)-helpful (R2) experiments: R1 (-0.94732502, 1.92034349), R2: (-0.01500361, 1.40736504).

For the training set, we use all the train set from hh-rlhf like this: load_dataset(hhrlhf_dataset_path, split='train'). For eval set, 
we use all the test set: load_dataset(hhrlhf_dataset_path, split='test'). You can check whether the read-team data is included when using the above loading strategy.

**Comment by XiaoPeiYao** [comment]
Hi, thanks for this amazing work and code open-sourcing!
I am also interested in your work. For the humor attribute, what are the mean and std to normalize?
Thanks

**Comment by YangRui2015** [comment]
> Hi, thanks for this amazing work and code open-sourcing! I am also interested in your work. For the humor attribute, what are the mean and std to normalize? Thanks

Hi, thank you for your interest. Unfortunately, I can’t access my previous dataset at the moment because the server is unavailable. However, you can generate the dataset and its statistics by running the following script: https://github.com/YangRui2015/RiC/blob/main/ric/prepare_dataset_with_rewards.py.

**Comment by XiaoPeiYao** [comment]
> > Hi, thanks for this amazing work and code open-sourcing! I am also interested in your work. For the humor attribute, what are the mean and std to normalize? Thanks
> 
> Hi, thank you for your interest. Unfortunately, I can’t access my previous dataset at the moment because the server is unavailable. However, you can generate the dataset and its statistics by running the following script: https://github.com/YangRui2015/RiC/blob/main/ric/prepare_dataset_with_rewards.py.

Thanks for the answer!
